package.json

```diff
@@ -10,44 +10,29 @@
     "url": "git://github.com/bigcommerce/stencil-utils.git"
   },
   "scripts": {
-    "test": "npx grunt eslint && npx grunt karma",
-    "lint": "npx grunt lint",
-    "build": "npx grunt"
+    "build": "webpack --config webpack.conf.js",
+    "test": "jest",
+    "test-with-coverage": "jest --coverage --verbose",
+    "lint": "eslint .",
+    "lint--fix": "eslint . --fix"
   },
   "dependencies": {
     "eventemitter3": "^4.0.4",
     "whatwg-fetch": "^3.4.0"
   },
   "devDependencies": {
-    "babel-core": "6.5.2",
-    "babel-eslint": "4.1.0",
-    "babel-loader": "^7.1.2",
-    "babel-plugin-dynamic-import-webpack": "^1.0.2",
-    "babel-plugin-lodash": "^3.3.2",
-    "babel-plugin-transform-regenerator": "^6.26.0",
-    "babel-preset-env": "^1.6.1",
-    "babel-preset-es2015": "^6.6.0",
-    "es6-shim": "^0.35.0",
-    "eslint": "2.4.0",
-    "eslint-config-airbnb": "6.0.2",
-    "grunt": "0.4.5",
-    "grunt-cli": "^1.3.2",
-    "grunt-eslint": "18.0.0",
-    "grunt-karma": "2.0.0",
-    "jasmine-core": "2.8.0",
-    "karma": "1.7.1",
-    "karma-babel-preprocessor": "7.0.0",
-    "karma-coverage": "1.1.1",
-    "karma-es6-shim": "^1.0.0",
-    "karma-jasmine": "^1.1.0",
-    "karma-phantomjs-launcher": "1.0.4",
-    "karma-sourcemap-loader": "^0.3.7",
-    "karma-verbose-reporter": "0.0.6",
-    "karma-webpack": "^2.0.6",
-    "load-grunt-config": "0.19.2",
-    "npx": "^10.2.0",
-    "time-grunt": "1.4.0",
-    "webpack": "^3.8.1"
+    "@babel/core": "^7.11.4",
+    "@babel/preset-env": "^7.11.0",
+    "babel-eslint": "^10.1.0",
+    "babel-jest": "^26.3.0",
+    "babel-loader": "^8.1.0",
+    "eslint": "^7.2.0",
+    "eslint-config-airbnb-base": "^14.2.0",
+    "eslint-loader": "^4.0.2",
+    "eslint-plugin-import": "^2.22.0",
+    "jest": "^26.4.2",
+    "webpack": "^4.44.1",
+    "webpack-cli": "^3.3.12"
```

Comment:
why do we need the cli package ?

Reply:
For running build from cli https://webpack.js.org/guides/installation/
<img width="740" alt="Screenshot 2020-09-11 at 13 23 27" src="https://user-images.githubusercontent.com/68893868/92911931-fdfccf00-f431-11ea-85bf-fb7150149ad8.png">